source/draw/AbstractContext.ooc

```diff
@@ -36,4 +36,6 @@ AbstractContext: abstract class {
 	alignWidth: virtual func (width: Int, align := AlignWidth Nearest) -> Int { width }
 	update: abstract func
 	isAligned: virtual func (width: Int) -> Bool { true }
+	_createContext := static func -> This { CpuContext new() }
```

Comment:
Oh, right. Got it. We don't use reassignable func members a lot, so I wasn't even sure what this did. Thanks.